network/fabrics_test.go

```diff
@@ -0,0 +1,875 @@
+package network_test
+
+import (
+	"context"
+	"net/http"
+	"path"
+	"strings"
+	"testing"
+
+	"strconv"
+
+	"io/ioutil"
+
+	"github.com/joyent/triton-go/network"
+	"github.com/joyent/triton-go/testutils"
+	"github.com/pkg/errors"
+)
+
+var (
+	fakeFabricVlanId      = 2
+	deleteFabricErrorType = errors.New("unable to delete fabric")
+	deleteVLanErrorType   = errors.New("unable to delete vlan")
+	createVLanErrorType   = errors.New("unable to create vlan")
+	createFabricErrorType = errors.New("unable to create fabric")
+	getVlanErrorType      = errors.New("unable to get vlan")
+	getFabricErrorType    = errors.New("unable to get fabric")
+	updateVlanErrorType   = errors.New("unable to update vlan")
+	listVlansErrorType    = errors.New("unable to list vlans")
+	listFabricsErrorType  = errors.New("unable to list fabrics")
+)
+
+func TestDeleteFabric(t *testing.T) {
+	networkClient := MockNetworkClient()
+
+	do := func(ctx context.Context, nc *network.NetworkClient) error {
```

Comment:
`s/do/testCase/g` ?  Subjectively, the code reads better to me with `testCase` as the name.